projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -173,6 +184,45 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
     event.stopPropagation();
   }
 
+  onSpace(event: UIEvent): void {
```

Comment:
It could be nice to have JSDoc comments to explain what these new methods do.

Reply:
Comments added :)